core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooCache.java

```diff
@@ -239,7 +239,7 @@ public ZooCache(ZooReader reader, Watcher watcher) {
     this.externalWatcher = watcher;
   }
 
-  private abstract class ZooRunnable<T> {
+  public abstract class ZooRunnable<T> {
```

Comment:
Why change this to public?

Reply:
It's now private again. I was trying to use this outside of the class but didn't end up heading in that direction.